src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: $border-radius;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: $primary-color;
+    font-size: $font-size-small;
+    line-height: 18px;
+    font-weight: 500;
+    text-align: left;
+  }
+
+  &__link {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 0;
+    text-decoration: none;
+  }
+
+  &__product-code {
+    margin: 4px 0 0;
+    line-height: 14px;
+    font-size: 10px;
+    color: $secondary-color;
+    text-align: left;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-title {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    font-size: $font-size-large;
```

Comment:
The variable `$font-size-large` is used here. Make sure it is defined in your SCSS files to avoid compilation issues.